linux_os/guide/services/ldap/openldap_client/enable_ldap_client/rule.yml

```diff
@@ -25,7 +25,7 @@ identifiers:
 
 references:
     disa: "1453"
-    nist: AC-17(a),AC-17(2),CM-7
+    nist: AC-17(a),AC-17(2),CM-7,SC-12(3)
```

Comment:
SC-12(3) | CRYPTOGRAPHIC KEY ESTABLISHMENT AND MANAGEMENT \| ASYMMETRIC KEYSThe organization produces, controls, and distributes asymmetric cryptographic keys using [Selection: NSA-approved key management technology and processes; approved PKI Class 3 certificates or prepositioned keying material; approved PKI Class 3 or Class 4 certificates and hardware security tokens that protect the user's private key].
-- | --


How does enabling LDAP client meet the crypto key establishment requirements?